plugin/test-keeper/plugin/test_checker.go

```diff
@@ -0,0 +1,60 @@
+package plugin
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/arquillian/ike-prow-plugins/git"
+)
+
+type TestChecker struct {
+	log *logrus.Entry
+}
+
+const configFile = "test-keeper.yaml"
+
+func (checker *TestChecker) checkTests(pr *git.PullRequest) (string, string, error) {
+
+	matchers := loadMatchers(pr)
+
+	checker.log.Infof("Checking for tests")
+	files, e := pr.GetAffectedFiles()
+	if e != nil {
+		return "", "", e
+	}
+
+	for _, matcher := range matchers {
+		for _, file := range files {
+			checker.log.Infof("%q: %q", file.Name, file.Status)
+
+			if matcher.isTest(file.Name) {
+				return "success", "There are some tests :)", nil
+			}
+		}
+	}
+
+	return "failure", "No tests in this PR :(", nil
+}
+
+func loadMatchers(pr *git.PullRequest) []TestMatcher {
+	var matchers []TestMatcher
+
+	config := pr.GetFile(configFile)
+	if config != nil {
+		matcher := loadMatcherFromConfig(config)
+		if matcher.TestRegex != "" {
+			matchers = append(matchers, matcher)
+		}
+	}
+
+	if len(matchers) == 0 {
+		languages, e := pr.GetRepoLanguages()
+		if e != nil {
+			matchers = append(matchers, defaultMatcher)
+		} else {
+			matchers = loadTestMatchers(languages)
+		}
+		logrus.Info("Using test matchers: ", matchers)
+	} else {
+		logrus.Info("Using test matcher from file: ", matchers)
```

Comment:
this is not for testing purposes - this is for transparency to know which matchers the test keeper is using.

Reply:
But why do we need this information logged in the runtime?

Reply:
why do we need to log anything - any information? For debug purposes/for transparency. This is the case that I see as useful to log info which matcher(s) is/are used.

Reply:
If you cover it with tests there is no need for printing this information out, really. If we need that for debugging purposes let's set it to `debug`.

Reply:
Also, this is going to be the service for multiple repositories - not sure if we are interested or even care to log it. this will become a noise fairly quick. I would log important information and errors only.